src/main/webapp/panelStyle.js

```diff
@@ -103,9 +102,20 @@ function createTextListElement(inputElementId, listElementId) {
     inputElement.value = '';
     return;
   }
-  phrase = phrase.replace(',', '');
+  phrase = phrase.split(',').join('');
```

Comment:
Merge issue?

Reply:
Yeah merge issue - thanks for catching this!